beta/src/pages/learn/react-developer-tools.md

```diff
@@ -30,26 +29,26 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal:
+अगला टर्मिनल से डेवलपर टूल खोलें:
 ```bash
 react-devtools
 ```
 
-Then connect your website by adding the following `<script>` tag to the beginning of your website's `<head>`:
+फिर अपनी वेबसाइट के `<head>` की शुरुआत में निम्न `<script>` टैग जोड़कर अपनी वेबसाइट कनेक्ट करें:
 ```html {3}
 <html>
   <head>
     <script src="http://localhost:8097"></script>
 ```
 
-Reload your website in the browser now to view it in developer tools.
+अपनी वेबसाइट को डेवलपर टूल में देखने के लिए उसे अभी ब्राउज़र में पुनः लोड करें।
```

Comment:
`पुनः लोड` => `रीलोड`

let me know if you have any other better suggestion for this.